src/stampy_nlp/settings.py

```diff
@@ -5,6 +5,8 @@
 load_dotenv()
 
 
+LOG_LEVEL: str = os.getenv('LOG_LEVEL', 'INFO')
```

Comment:
or should the default level be debug? That seems excessive to have on the server

Reply:
Info is fine. You're right that debug's excessive. Mainly just want the user queries.